### PR TITLE
Added OpenmediaVault; EZ-WifiBroadcast; DroneBridge

### PR DESCRIPTION
**Pull request by careyer**
all projects offer ready made OS images for direct download

# Please make sure all below checkboxes have been checked before submitting your PR.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
	- [x] includes a valid (https) link,
	- [x] includes a concise and on-topic description,
	- [x] mentions if there is only support some devices,
	- [x] is not a duplicate,
	- [x] has been added at the bottom of the appropriate category,
	- [x] is in the form of **Named Link - Description, seperated by commas.**
	- [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Contributions *very welcome* but first see [Contributing](#contributing)
 - [Hass.io](https://home-assistant.io/hassio/installation/) - Home automation operating system/application for embedded device, also available standalone.
 - [Android Things](https://developer.android.com/things/hardware/raspberrypi.html) - Build connected devices for a wide variety of consumer, retail, and industrial applications. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
 - [Gladys Project](https://gladysproject.com/en/) - Gladys, Your home assistant. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
+- [OpenMediaVault](https://www.openmediavault.org/) - OpenMediaVault is the next generation network attached storage (NAS) solution based on Debian Linux. It contains services like SSH, (S)FTP, SMB/CIFS, DAAP media server, RSync, BitTorrent client and many more. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
```
Comment (thibmaek) [inline README.md]: Please correct this one to not use two sentences, but use a more concise sentence (separated by comma)

File: README.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ Contributions *very welcome* but first see [Contributing](#contributing)
...
+- [EZ-WifiBroadcast](https://github.com/bortek/EZ-WifiBroadcast/wiki) - Affordable Wireless Digital HD Video Transmission made easy!. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png) ![Supports Raspberry Pi Zero](/media/badges/rpi-0.png)
```
Comment (thibmaek) [inline README.md]: As punctuation please use a period (.) here and not a combination of exclamation mark + period (!.)

File: README.md [hunk 3]
```diff
@@ -53,6 +53,10 @@ Contributions *very welcome* but first see [Contributing](#contributing)
...
+- [DroneBridge](https://github.com/seeul8er/DroneBridge) - A WifiBroadcast extension to make Betaflight/Cleanflight/iNAV/MavLink based flight controllers more usable for aerial photography. Intended to be a real alternative to DJI Lightbridge and other similar systems. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
```
Comment (thibmaek) [inline README.md]: Please correct this one to not use two sentences, but use a more concise sentence (separated by comma)

**Comment by thibmaek** [comment]
This PR looks good, aside from a few stylistic issues.

**Comment by thibmaek** [comment]
Cool, Thanks!

**Comment by careyer** [comment]
You're welcome! :-)

**Pull request merged**